### PR TITLE
Calculator Test Logic and Pages

### PR DESCRIPTION
**Pull request by Alexoid1**
- Add configuration of test
- Add test for pages with snapshot of components
- Add unit test for logic

### CODE REVIEW

File: src/test/operate.test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  });
+  it('should return the rest of numbers', () => {
+    const newData = operate(5, 20, '-');
+    expect(newData).toEqual('-15');
+  });
+  it('should return the multiplication of numbers', () => {
+    const newData = operate(5, 20, 'x');
+    expect(newData).toEqual('100');
+  });
+  it('should return the multiplication of numbers', () => {
```
Comment (bellom) [inline src/test/operate.test.js]: I think it should be `  it('should return the division of numbers', () => {`

File: src/test/operate.test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    const newData = operate(20, 5, '÷');
+    expect(newData).toEqual('4');
+  });
+  it('should return the multiplication of numbers', () => {
```
Comment (bellom) [inline src/test/operate.test.js]: I think it should be `  it('should return the percentage of numbers', () => {`

**Review by bellom — APPROVED** [review]
## Hi @Alexoid1  :clap: 

##### You have done wonderfully well, you passed all checklist for this project :+1:
### What a great design! 👍 
 
# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck with your next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun J. Bello**
**_Slack Handle - @sean bellom_**

File: src/test/operate.test.js (hunk 1 above)
Comment (Alexoid1) [reply src/test/operate.test.js]: thanks

**Pull request merged**